lib/prefixed_ids.rb

```diff
@@ -123,4 +124,18 @@ def to_param
       _prefix_id.encode(id)
     end
   end
+
+  module Exists
+    extend ActiveSupport::Concern
+
+    class_methods do
+      def exists?(id)
+        if _prefix_id.present? && id.is_a?(String)
+          super(_prefix_id.decode(id))
+        else
+          super(id)
```

Comment:
I guess this can also just be super and forward along the arguments automatically.
```suggestion
          super
```